src/components/GameComponent.vue

```diff
@@ -222,7 +222,8 @@ function setAnswers(game: Game) {
   tempAnswers.push(rightAnswer);
   // Sort answers so they are displayed in a consistent order.
   // Otherwise, the order will change on every update from the backend.
-  sortedAnswers.value = tempAnswers.sort();
+  tempAnswers.sort();
+  sortedAnswers.value = tempAnswers;
```

Comment:
`.sort()` does not modifiy the contents of tempAnswers, but instead returns a completely new sorted Array. With this change, the answers will no longer be sorted.

Reply:
nevermind, this works